.github/scripts/git-get-shared-history.sh

```diff
@@ -20,6 +20,13 @@ set -euxo pipefail
 
 # We can always use some more history
 git -c protocol.version=2 fetch --deepen=100
+
+# Find out if the PR is coming from a fork
```

Comment:
@paulirish how do you feel about this skipping this check entirely if it's from a fork for now instead and we can sort out the fix later?

Reply:
that works for me.. though determining that would probably have required almost this much iteration as well...    so i think it's all moot.

ALL MOOT!